electron/electron.js

```diff
@@ -30,30 +30,38 @@ const debug = require("electron-debug");
 const logger = require("electron-log");
 const setupListeners = require("./execution");
 const buildMenu = require("./menu");
-
 // For dev
 unhandled({ logger: err => logger.error(err) });
 debug({ enabled: true, showDevTools: false });
 
 const BUILD_DIR = join(__dirname, "..", "build");
 
+// We can't read from the `env` file within `services` here
+// so we must access the process env directly
+const IS_TEST = process.env.NODE_ENV === "test";
+const TEST_PORT = process.env.TEST_PORT;
+
 async function createWindow() {
   const win = new BrowserWindow({
     width: 1100,
     height: 700,
     minHeight: 500,
     minWidth: 800,
     webPreferences: {
-      devTools: isDev,
+      devTools: isDev || IS_TEST,
       nodeIntegration: true,
       contextIsolation: false,
       nativeWindowOpen: true,
     },
   });
 
-  isDev
-    ? win.loadURL("http://localhost:3000")
-    : win.loadFile(join(BUILD_DIR, "index.html"));
+  if (isDev && !IS_TEST) {
+    win.loadURL("http://localhost:3000");
```

Comment:
I'm just reflecting here, might be a nice future enhancement for this to not be hardcoded too. Out of scope for this patch of course.